src/unistd.rs

```diff
@@ -1644,8 +1644,18 @@ mod linux {
         Errno::result(res).map(drop)
     }
 
+    /// Replaces the current process image with a new one.
+    /// ([see exec(3)](http://man7.org/linux/man-pages/man3/exec.3.html))
+    ///
+    /// * `filename`: filename of executable to run
```

Comment:
Can you give a title or quick sentence to introduce this bulleted list? It doesn't read nicely just being a stand alone list.